metron-interface/metron-alerts/src/app/alerts/alerts-list/alerts-list.component.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { RouterTestingModule } from '@angular/router/testing';
+import { SearchService } from 'app/service/search.service';
+import { UpdateService } from 'app/service/update.service';
+import { ConfigureTableService } from 'app/service/configure-table.service';
+import { AlertsService } from 'app/service/alerts.service';
+import { ClusterMetaDataService } from 'app/service/cluster-metadata.service';
+import { SaveSearchService } from 'app/service/save-search.service';
+import { MetaAlertService } from 'app/service/meta-alert.service';
+import { GlobalConfigService } from 'app/service/global-config.service';
+import { DialogService } from 'app/service/dialog.service';
+import { Observable } from 'rxjs';
+import { Filter } from 'app/model/filter';
+
+fdescribe('AlertsListComponent', () => {
+
+  let component: AlertsListComponent;
+  let fixture: ComponentFixture<AlertsListComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      schemas: [ NO_ERRORS_SCHEMA ],
+      imports: [
+        RouterTestingModule.withRoutes([]),
+      ],
+      declarations: [
+        AlertsListComponent,
+      ],
+      providers: [
+        { provide: SearchService, useClass: () => { return {} } },
+        { provide: UpdateService, useClass: () => { return {
+          alertChanged$: new Observable(),
+        } } },
+        { provide: ConfigureTableService, useClass: () => { return {
+          getTableMetadata: () => new Observable(),
+          tableChanged$: new Observable(),
+        } } },
+        { provide: AlertsService, useClass: () => { return {} } },
+        { provide: ClusterMetaDataService, useClass: () => { return {
+          getDefaultColumns: () => new Observable(),
+        } } },
+        { provide: SaveSearchService, useClass: () => { return {
+          loadSavedSearch$: new Observable(),
+        } } },
+        { provide: MetaAlertService, useClass: () => { return {
+          alertChanged$: new Observable(),
+        } } },
+        { provide: GlobalConfigService, useClass: () => { return {
+          get: () => new Observable(),
+        } } },
+        { provide: DialogService, useClass: () => { return {} } },
+      ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(AlertsListComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should exist', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should set default query time range', () => {
+    expect(component.selectedTimeRange instanceof Filter).toBeTruthy();
+    expect(component.selectedTimeRange.value).toBe('last-15-minutes');
+  });
+
+  it('default query time range from date should be sat', () => {
+    expect(component.selectedTimeRange.dateFilterValue.fromDate).toBeTruthy();
+  });
+
+  it('default query time range to date should be sat', () => {
```

Comment:
I think you have a typo here. Did you mean, "default query time range to date should be set"?